Quotient/room.cpp

```diff
@@ -488,7 +491,14 @@ Room::Room(Connection* connection, QString id, JoinState initialJoinState)
         connect(this, &Room::beforeDestruction, this, [id, connection] {
             connection->database()->clearRoomData(id);
         });
+        //TODO: delete from database when leaving
     }
+
+    //TODO how do we deal with encrypted events? store encrypted or unencrypted? if unencrypted, we need to also store encrypted to allow clients to view the encrypted event
+    QMetaObject::invokeMethod(this, [=, this](){
+        d->loadFromDb(50);
```

Comment:
(note to myself:) we don't want this, as it would load messages for all rooms on startup